browserstack.json

```diff
@@ -4,15 +4,31 @@
 	"test_framework": "qunit",
 	"test_path": [
 		"test/index.html",
+		"test/startError.html",
 		"test/logs.html",
-		"test/amd.html"
+		"test/amd.html",
+		"test/autostart.html"
 	],
 	"browsers": [
 		"chrome_previous",
 		"chrome_current",
 		"firefox_previous",
 		"firefox_current",
+		"opera_previous",
 		"opera_current",
+		"safari_5_1",
+		{
+			"browser": "safari",
+			"browser_version": "6.0"
+		},
+		{
+			"browser": "safari",
+			"browser_version": "6.2"
```

Comment:
Also, note that you're not testing Safari 7.0. This is temporarily impossible due to a mishap on BrowserStack's part but I hope it will be resolved soon.